pub-kube-config-files/publish-availability-monitor.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: publish-availability-monitor
+  labels:
+    app: publish-availability-monitor
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: publish-availability-monitor
+  template:
+    metadata:
+      labels:
+        app: publish-availability-monitor
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: publish-availability-monitor
+        image: coco/publish-availability-monitor:v0.24.17-k8s-version-rc7
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 512Mi
+        env:
+        - name: KAFKA_TOPIC
+          value: "PreNativeCmsPublicationEvents"
+        - name: KAFKA_PROXY_HOST
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: kafka.proxy.url
+        - name: QUEUE_ADDR
+          value: "http://kafka:8082"
+        - name: CONTENT_URL
+          value: "/__document-store-api/content/"
+        - name: INTERNAL_COMPONENTS_URL
+          value: "/__document-store-api/internalcomponents/"
```

Comment:
the host is dynamically injected into the app through watching mechanism and it is taken from `monitoring-configs` configMap (see entry with key `read-urls`). Note that we may have multiple read environments (e.g. pre-prod-uk, pre-prod-us). We don't need to specify the port since we reach the read endpoints via varnish